syntax = "proto3";

package health;

// Response message for Health
message HealthResponse {
    string message = 1;
}
